vfs/file_test.go

```diff
@@ -17,20 +24,74 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func fileCreate(t *testing.T, mode vfscommon.CacheMode) (r *fstest.Run, vfs *VFS, fh *File, item fstest.Item, cleanup func()) {
+func fileCreateLinks(t *testing.T, mode vfscommon.CacheMode, links bool) (r *fstest.Run, vfs *VFS, fh *File, item fstest.Item, lh *File, linkItem fstest.Item, cleanup func()) {
 	opt := vfscommon.DefaultOpt
 	opt.CacheMode = mode
 	opt.WriteBack = writeBackDelay
+	opt.Links = links
 	r, vfs, cleanup = newTestVFSOpt(t, &opt)
 
 	file1 := r.WriteObject(context.Background(), "dir/file1", "file1 contents", t1)
 	r.CheckRemoteItems(t, file1)
 
-	node, err := vfs.Stat("dir/file1")
+	node, err := vfs.Stat(file1.Path)
 	require.NoError(t, err)
 	require.True(t, node.Mode().IsRegular())
 
-	return r, vfs, node.(*File), file1, cleanup
+	// For some reason this does not works - Stat fails and return file not found ?
```

Comment:
???

Reply:
Yep, never had the code commented working, if you have an idea ?
I can also just remove it as well entirely, but I would prefer to understand why it did not worked.
Initially this function was expecting to create a regular file and a symlink file, but for some reason trying to add more than one file failed and I did not found why.

Reply:
Closing as part of the old trashed code